extensions/objects/templates/FilepathEditor.js

```diff
@@ -73,6 +75,21 @@ oppia.directive('filepathEditor', [
                  data.metadata.savedUrl.length > 0;
         };
 
+        $scope.getImageSizeHelp = function() {
+          var imageWidth = $scope.data.metadata.originalWidth;
+          if ($scope.resizeRatio === 1 &&
+              imageWidth > $scope.MAX_OUTPUT_IMAGE_WIDTH) {
+            return 'Even at 100%, the image is being automatically downsized ' +
```

Comment:
Prefer: 'This image has been automatically downsized to ensure that it will fit in the card.'